networkx/algorithms/link_prediction.py

```diff
@@ -340,9 +340,9 @@ def predict(u, v):
                 raise nx.NetworkXAlgorithmError("Self loops are not supported")
             path_len = spl[u].get(v, inf)
 
-            return alpha * sum(1 for _ in nx.common_neighbors(G, u, v)) + (
-                1 - alpha
-            ) * (G.number_of_nodes() / path_len)
+            return alpha * len(nx.common_neighbors(G, u, v)) + (1 - alpha) * (
+                G.number_of_nodes() / path_len
+            )
```

Comment:
```suggestion
            n_nbrs = len(nx.common_neighbors(G, u, v))
            return alpha * n_nbrs + (1 - alpha) * len(G) / path_len
```
Is this easier to read?